app/src/main/java/com/io/movio/ui/movielist/MovieListFragment.kt

```diff
@@ -40,10 +40,7 @@ class MovieListFragment : Fragment(), MoviesAdapter.ItemOnClickListener {
         }
     }
 
-    override fun onItemClick(id: Int) {
-        requireActivity().supportFragmentManager.beginTransaction()
-            .replace(R.id.fragment_container, MovieDetailFragment.newInstance(id))
-            .addToBackStack(javaClass::class.java.name)
-            .commit()
-    }
+    override fun onItemClick(id: Int) = Navigation
+        .findNavController(binding.recyclerView)
```

Comment:
it's not wrong to use any view in the binding, but it's more readable if you use - `requireview()`